docs/tutorials/losses_focal.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>Same, this block is unnecessary :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>